### ISSUE TITLE
HuggingFace: Add Pipeline Warmup on Startup

### ISSUE THREAD

**Issue by mikesvoboda** [mikesvoboda/nemotron-v3-home-security-intelligence#4377]
## Overview

Warm up inference pipeline during startup to eliminate first-request latency.

## Solution

```python
async def warmup_pipeline(pipeline, warmup_samples: int = 3):
    dummy_input = 'Warmup text for model initialization.'
    
    for _ in range(warmup_samples):
        with torch.no_grad():
            _ = pipeline(dummy_input, max_new_tokens=10)
    
    torch.cuda.synchronize()
    torch.cuda.empty_cache()
```

## Implementation

* Add warmup to model loading
* Run multiple samples to stabilize CUDA
* Clear cache after warmup

## Impact

Eliminate first-request latency spike
